airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -84,7 +84,7 @@
         "order": 6,
         "format": "date"
       },
-      "custom_queries": {
+      "custom_queries_array": {
```

Comment:
Why are we renaming the field?

Reply:
This property was renamed for backwards compatibility. When falling back to the previous version old property will be used.